### DISCUSSION TITLE
Do we need to submit generated checkstyle?

### DISCUSSION THREAD

**Discussion by DelusionArun** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#54] [Q&A] [answered]
After a few problems with my older submissions, I downloaded a zip from my git, ran it one time, and submitted the new zip to CASS a few days ago. I now realize that I forgot to have checkstyle generated. 
Should I generate a checkstyle and re-submit or will checkstyle be generated/checked by TA?

**Comment by Derppening** [comment, marked as answer]
I will generate the CheckStyle report on your behalf.

Same applies to any generated artifacts, you do not need to submit JUnit test reports, CheckStyle reports, your compiled classes etc.
